src/debugger/DebuggerTcp.cpp

```diff
@@ -344,6 +349,28 @@ void DebuggerTcp::init(const char* options, Context* context)
 
     ESCARGOT_LOG_INFO("Waiting for client connection 0.0.0.0:%hd\n", port);
 
+    while (true) {
+        struct pollfd fd[1];
+        fd[0].fd = serverSocket;
+        fd[0].events = POLLIN;
```

Comment:
Why don't you move them out of the loop?

Reply:
Good point!